src/test/java/bio/terra/service/filedata/google/firestore/ArrayMultiFileLoadTest.java

```diff
@@ -282,20 +268,19 @@ public void arrayMultiFileLoadFailRetryTest() throws Exception {
     TableResult queryLoadHistoryTableResult = queryLoadHistoryTable(columnsToQuery);
     for (FieldValueList item : queryLoadHistoryTableResult.getValues()) {
       String state = item.get(0).getStringValue();
```

Comment:
It might be better to convert to `BulkLoadFileState` here to avoid the `toString()`s below
```suggestion
      BulkLoadFileState state = BulkLoadFileState.valueOf(item.get(0).getStringValue());
```

Reply:
Nice, and after doing that it felt natural to put in a switch statement to handle the assertions.